### PR TITLE
New package: tldd

### PR DESCRIPTION
**Pull request by rmsds**
* https://gitlab.com/miscripts/tldd

### CODE REVIEW

File: var/spack/repos/builtin/packages/tldd/package.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+class Tldd(Package):
+    """A program similar to ldd(1) but showing the output as a tree."""
+
+    homepage = "https://gitlab.com/miscripts/tldd"
+    url      = "https://gitlab.com/miscripts/tldd"
+    git      = "https://gitlab.com/miscripts/tldd.git"
+
+    version('master', branch='master')
+
+    depends_on('pstreams')
```
Comment (healther) [inline var/spack/repos/builtin/packages/tldd/package.py]: @0.8.0:

File: var/spack/repos/builtin/packages/tldd/package.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+class Tldd(Package):
+    """A program similar to ldd(1) but showing the output as a tree."""
+
+    homepage = "https://gitlab.com/miscripts/tldd"
+    url      = "https://gitlab.com/miscripts/tldd"
+    git      = "https://gitlab.com/miscripts/tldd.git"
+
+    version('master', branch='master')
```
Comment (healther) [inline var/spack/repos/builtin/packages/tldd/package.py]: not sure if it's worth the additional work, but you could add a commit hash based version, which would make this package prefetchable

File: var/spack/repos/builtin/packages/tldd/package.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    depends_on('pstreams')
+
+    def install(self, spec, prefix):
```
Comment (healther) [inline var/spack/repos/builtin/packages/tldd/package.py]: this method shouldn't be necessary.

Comment (rmsds) [reply var/spack/repos/builtin/packages/tldd/package.py]: It didn't work out of the box unless I specified the PREFIX variable. 
Or maybe are you suggesting something like what is done for [p7zip](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/p7zip/package.py) for example? Make it a `MakefilePackage` at redefine the `install_targets()`?

Comment (rmsds) [reply var/spack/repos/builtin/packages/tldd/package.py]: Even if I make it a `MakefilePackage` it still doesn't work out of the box, it tries to install to `/usr/local/bin`, which comes from the Makefile: `PREFIX := /usr/local`. 

From the Environment Variables section in the documentation for [MakeFilePackage](https://spack.readthedocs.io/en/latest/build_systems/makefilepackage.html?highlight=makefilepackage#environment-variables) I now understand better what the options are. Depending on how the Makefile is written, you can either set an environment variable; pass a parameter (the tldd case); or edit the Makefile.

As the `:=` assignment operator is used for PREFIX I need to pass a parameter to the `make install`.

**Pull request merged**